### ISSUE TITLE
Keyboard shortcut CTRL+ALT+E Run From Line to End is not working

### ISSUE THREAD

**Issue by wphampton** [rstudio/rstudio#8404]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.4.1029
    OS Version      : Windows 7
    R Version       : 4.0.3

### Steps to reproduce the problem
1. Open an .R script
2. Move cursor somewhere in middle of script
3. Type CTRL+ALT+E

### Describe the problem in detail
Using the keyboard shortcut CTRL+ALT+E does not run any code. However, if I navigate to the menu Code > Run Region > Run From Line to End, it does run the code as expected.

### Describe the behavior you expected
I expected the behavior to be the same whether I used the keyboard shortcut or selected the menu item.

- [ x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [ x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by kevinushey** [comment]
Thanks for taking the time to file a bug report! Unfortunately, I'm not able to reproduce -- this seems to work for me without issue. Can you share any more details?

**Comment by wphampton** [comment]
Sheesh, I don't have an explanation yet but I think let's call this not a bug and close for now. I can't get CTRL + ALT + E to work from my workstation within RStudio desktop or when I connect to RStudio server, even when I use the built-in  Windows On-Screen Keyboard. _However_, when I remote desktop to another computer and do the same it does work as designed. So, I'm thinking there is something swallowing up this key combination within my Windows 7 OS but somehow not when I remote desktop to another computer. Strange, but I'll report back and take my lumps once I figure out what I'm missing here. Thanks for your reply.

**Comment by wphampton** [comment]
This is not an issue with RStudio. Our organization recently pushed Avaya one-X Communicator to our workstations and it has a global hotkey binding of Alt + Control + E to end a phone call. I used Microsoft Spy++ to track down the global WM_HOTKEY event as I had I idea where it was going and found it was bound to Module Name "ONEXCUI".  I removed the Hot Key from  Avaya one-X Communicator and RStudio CTRL + ALT + E now functions correctly!

I used this article (https://stackoverflow.com/a/43645062/2184705) to learn how to track it down. I installed Microsoft Visual Studio 2019 Community and then specifically selected to install the "C++ profiling tools" under the Individual Components so that Spy++ was installed.

Sorry for the not-bug report. -Wes

![image](https://user-images.githubusercontent.com/1633835/99403058-0bae5500-28b8-11eb-8ea5-5efc2a453148.png)
